operators/src/processing/point_in_polygon/tester.rs

```diff
@@ -90,6 +99,39 @@ impl<'a> PointInPolygonTester<'a> {
         }
     }
 
+    fn precalculate_polygon_bounds(
+        feature_offsets: &'a [i32],
+        polygon_offsets: &'a [i32],
+        ring_offsets: &'a [i32],
+        coordinates: &'a [Coordinate2D],
+    ) -> Vec<Vec<BoundingBox2D>> {
+        let mut multi_poly_bounds = Vec::with_capacity(feature_offsets.len() - 1);
+
+        for (feature_start_index, feature_end_index) in
+            two_tuple_windows(feature_offsets.iter().map(|&c| c as usize))
```

Comment:
why do you not use the iterators on the MultiPolygonCollection? Like `polygons.geometries().for_each()` or something like that. you should be able to get the coordinates this way too.

Reply:
i thought that it would be a good idea to reuse the slices already created. However, i rebuild it using the iterators. [81ff742](https://github.com/geo-engine/geoengine/pull/792/commits/81ff742edf53e417c4c48d9c70684e9ba3889e51)